spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +62,25 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with path to sqlite3 file' do
+      it 'returns a dataframe' do
+        result = Daru::IO::SqlDataSource.make_dataframe(db_name, query)
+        # result = Daru::DataFrame.from_sql(db_name, query)
+        expect(result).to be_a(Daru::DataFrame)
+        expect(result.nrows).to eq(2)
+        expect(result.row[0][:id]).to eq(1)
+        expect(result.row[0][:name]).to eq('Homer')
+      end
```

Comment:
Let's make a proper "modern" specs for a new functionality (like http://betterspecs.org/ modern). With subject and `its(...)` and stuff. For entire daru codebase it is a long road to better specs, but for new ones, it is reasonable to have them in decent form from the very beginning.